scripts/ci.sh

```diff
@@ -65,6 +65,10 @@ GENERAL_OPTIONS="--verbose --deployer 0.0.0.0"
 COMMON_OPTIONS="--topology topology-min.yaml"
 CARDANO_COMPONENTS="Nodes ${WITH_EXPLORER:+Explorer} ${WITH_REPORT_SERVER:+ReportServer}"
 
+nix-build default.nix -A cardano-sl-tools -o cardano-sl-tools
```

Comment:
Hmm, why is this needed? Shouldn't the code just use nix-shell if it needs these tools?

Reply:
the CI runs `iohk-ops` without a nix-shell, and was relying on the fact that it was `makeWrapper'`d to have cardano-tools in `PATH`

and the whole point of this PR is to not depend on cardano when building io

Reply:
When does CI need it exactly?